package.json

```
@@ -24,7 +24,7 @@
     "prepare": "husky || true"
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=18.18.0"
```

Unlike Content, this repo doesn't have a dependency on Yari. So we can use the minimum required `18.12.0`:
```suggestion
    "node": ">=18.12.0"
```